server/src/test/java/org/elasticsearch/common/settings/SettingTests.java

```diff
@@ -206,6 +206,10 @@ public void testValidateStringSetting() {
 
         public static boolean invoked;
 
+        @Override
+        public void validate(String value) {
```

Comment:
I think this method should also contain assertions and we should also assert that it is called in the right places.

Reply:
Yes, you're right, I'll have a look at it.

Reply:
Good catch! Fixed in 3e58473, thank you :+1: